src/App.jsx

```diff
@@ -1,16 +1,20 @@
-import { useState } from "react";
-import reactLogo from "./assets/react.svg";
-import viteLogo from "/vite.svg";
-import "./App.css";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Books from "./pages/Books";
+import Categories from "./pages/Categories";
+import Navbar from "./components/Navbar";
 
 function App() {
-	const [count, setCount] = useState(0);
-
-	return (
-		<>
-			<h1 className="text-red-600 capitalize">books library</h1>
-		</>
-	);
+  return (
+    <>
+      <BrowserRouter>
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Books />} />
```

Comment:
Seems our routes is not specifying an `index` routes. please look into this

Reply:
All changes made as required